{
  "modified": "2024-06-25T13:34:09Z",
  "published": "2024-06-25T13:34:09Z",
  "schema_version": "1.5.0",
  "id": "MAL-2024-4983",
  "summary": "Malicious code in colorsmore (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "colorsmore",
        "purl": "pkg:pypi/colorsmore"
      },
      "versions": [
        "5"
      ]
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "id": "RLMA-2024-03763",
        "import_time": "2024-06-28T02:48:40.717141461Z",
        "modified_time": "2024-06-25T13:34:09Z",
        "sha256": "6f88004e8b230c8bf848a31f5d76b321aa54d0fe2e29a4926941c74085101f54",
        "source": "reversing-labs",
        "versions": [
          "5"
        ]
      }
    ]
  }
}
